### PR TITLE
feat: configurable GeoJSON coordinate precision (default 6)

### LINKED ISSUE #7: Shorten WKT coordinates (limit decimals)
At the moment WKT coordinates have unnecessarily high precision.
lat/lon decimals could be shortened to 6 digits still without loosing actual precision:
```
POINT (135.27597373034055 34.68437899629821)
```
Not sure, if RGeo provides functionality to limit coordinates decimals.

### PR DESCRIPTION
**Pull request by dkastl**
Closes #7.

## Problem

GeoJSON output carried full floating-point precision (15-17 digits), e.g. `135.27597373034055`. For EPSG:4326 that is far beyond any real-world accuracy (6 places ≈ 0.11 m, 7 ≈ 1 cm) and it bloats the payload, which is noticeable for large polygons. RGeo's GeoJSON encoder offers no precision option and upstream declines to add one (the original concern in #7), so the plugin now rounds coordinates itself.

## Change

- **Setting**: new `geojson_precision` (default `6`, clamped to `0..15`) read via `RedmineGtt.geojson_precision`. Storage keeps full precision; only output is rounded, so nothing is lost.
- **PostGIS path**: `ST_AsGeoJson(geom, <precision>)` (the `maxdecimaldigits` argument). The precision is a clamped Integer, so the interpolation is injection-safe.
- **RGeo path** (`Conversions::GeomToJson`): rounds the numbers under each `coordinates` key after encoding, leaving `properties` untouched. This covers the map's issues-index GeoJSON, `.geojson` downloads, and single-feature output.
- **Settings UI**: a number field on the General tab, with en/ja/de strings.
- `geodata_for_print` (single-feature print, EPSG:3857 meters) is intentionally left as-is; it is a different unit and not the payload concern.

## Verification

- Live in a Redmine 6.1 dev instance with a high-precision point: both the RGeo path (`.geojson`, `.json`) and the PostGIS path (index `db_geojson`) output `[139.691701, 35.68953, 0.0]` at the default; setting precision to `2` yields `[139.69, 35.69]` on both. Settings field renders and round-trips.
- Ruby tests pass (unit issue/project/user + API integration + gtt_map + filter API: 30 runs, 305 assertions, 0 failures).

### Test note

`assert_equal_coordinates` previously compared at 5 decimals. Now that output is rounded to 6, comparing *below* the output precision double-rounds boundary values (e.g. `135.2528349` → `round(6)` `135.252835` → `round(5)` `135.25284`, whereas a direct `round(5)` is `135.25283`). The helper now compares at the output precision, and the "small geom changes" assertions use that rounding comparator instead of exact float equality.

### CODE REVIEW

File: test/test_helper.rb [hunk 1]
```diff
@@ -44,7 +44,12 @@ def assert_geojson(json)
   end
 
   def assert_equal_coordinates(a, b)
-    assert_equal a.flatten.map{|f|f.round 5}, b.flatten.map{|f|f.round 5}
+    # Compare at the precision GeoJSON output is rounded to. Comparing at a
+    # lower precision than the output uses would double-round boundary values
+    # (e.g. 135.2528349 -> round(6) 135.252835 -> round(5) 135.25284, while a
+    # direct round(5) gives 135.25283).
+    precision = RedmineGtt::DEFAULT_GEOJSON_PRECISION
+    assert_equal a.flatten.map{|f|f.round precision}, b.flatten.map{|f|f.round precision}
```
[the remark this answers is not in the document]
Comment (dkastl) [reply test/test_helper.rb]: Fixed in d225569: the helper now uses RedmineGtt.geojson_precision, so it follows the live setting (defaulting to 6 when unset) and stays correct if a test overrides the precision. Tests still green.
